sopel/modules/find_updates.py

```diff
@@ -35,14 +35,16 @@ def startup_version_check(bot, trigger):
     global startup_check_run
     if not startup_check_run:
         startup_check_run = True
-        check_version(bot)
+        check_version(bot, verify_ssl=bot.config.core.verify_ssl)
 
 
 @sopel.module.interval(wait_time)
-def check_version(bot):
+def check_version(bot, verify_ssl=True):
```

Comment:
Since we're passing the bot instance to this function, we should probably just pull verify_ssl from the config when we do the request instead of passing it as a function param.